src/Serialization/Fields.jl

```diff
@@ -48,3 +48,101 @@ function load_internal(s::DeserializerState, ::Type{gfp_fmpz_elem}, dict::Dict)
     F = load_type_dispatch(s, Nemo.GaloisFmpzField, dict[:parent])
     return F(load_type_dispatch(s, fmpz, dict[:data]))
 end
+
+################################################################################
+# SimpleNumField
+function save_internal(s::SerializerState,
+                       K::SimpleNumField{T}) where T
```

Comment:
I don't think the `where T` is needed here; the same applies to most of the later occurrences in your PR
```suggestion
                       K::SimpleNumField)
```